.babelrc

```diff
@@ -2,9 +2,12 @@
   "plugins": [
     "transform-object-rest-spread",
     "transform-flow-comments",
+    "transform-runtime",
     "add-module-exports"
   ],
   "presets": [
-    "es2015"
+    "es2015",
```

Comment:
I'll have to read up on `babel-preset-env`. Does it require a list of supported browsers to be created, and if so, where do we get that?